apps/docs/src/examples/animation/transitions/App.svelte

```diff
@@ -1,35 +1,44 @@
 <script lang="ts">
-	import { useTweakpane } from '$lib/useTweakpane'
-	import { Canvas } from '@threlte/core'
-	import Scene from './Scene.svelte'
-	import { buttonIdle, buttonWalk, buttonRun } from './state'
-
-	const { addButton, action } = useTweakpane()
-
-	addButton({
-		title: 'Idle',
-		onClick: () => {
-			$buttonIdle = !$buttonIdle
-		}
-	})
-
-	addButton({
-		title: 'Walk',
-		onClick: () => {
-			$buttonWalk = !$buttonWalk
-		}
-	})
-
-	addButton({
-		title: 'Run',
-		onClick: () => {
-			$buttonRun = !$buttonRun
-		}
-	})
+  import { Pane, Button } from 'svelte-tweakpane-ui'
+  import { Canvas } from '@threlte/core'
+  import Scene from './Scene.svelte'
+  import { buttonIdle, buttonWalk, buttonRun } from './state'
 </script>
 
-<div use:action />
+<Pane
+  title="Transitions"
+  position="fixed"
+>
+  <Button
+    title="Idle"
+    on:click={() => {
+      $buttonIdle = !$buttonIdle
+    }}
+  />
+  <Button
+    title="Walk"
+    on:click={() => {
+      $buttonWalk = !$buttonWalk
+    }}
+  />
+  <Button
+    title="Run"
+    on:click={() => {
+      $buttonRun = !$buttonRun
+    }}
+  />
+</Pane>
+
+<div>
```

Comment:
The wrapping of the `<Canvas>` element with a div + the styling adds quite a bit of boilerplate when accumulated across all examples. Was this done because of issues that popped up when switching over to iframes? I wonder if there's a way to solve the problem in one place to avoid having to add this to every example...